src/LanguageServer/Impl/Sources/DefinitionSource.cs

```diff
@@ -54,6 +55,29 @@ internal sealed class DefinitionSource {
                 }
 
                 var value = eval.GetValueFromExpression(expr);
+                if (value.IsUnknown()) {
+                    // If this is 'import A as B' A is not declared as a variable, so try modules.
+                    string moduleName = null;
+                    if (!string.IsNullOrEmpty(name)) {
+                        switch (statement) {
+                            case ImportStatement imp when imp.Names.Any(x => x?.MakeString() == name):
+                            case FromImportStatement fimp when fimp.Names.Any(x => x?.Name == name):
+                                moduleName = name;
```

Comment:
We should use `PathResolverSnapshot` here, otherwise it won't work for any complex cases.

Reply:
Yep, it doesn't work in more complex cases like `from a.b.c import ...` - `a.b.c` is not handled completely correctly (right click on `b` -> goto definition does not work`. Probably should file a separate issue to improve this.

Reply:
Ok, let's have a separate task for it. There were also couple of todos in the files. Do you want tasks for them as well?

Reply:
Depends on TODO. There is a bunch of placeholders to report future diagnostics, there is already one task for them - #581. Others may or may not common, so case by case.